core-java-modules/core-java-security-4/src/test/java/com/baeldung/enablessldebug/SSLDebugLoggerUnitTest.java

```diff
@@ -0,0 +1,31 @@
+package com.baeldung.enablessldebug;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+
+import org.junit.Test;
+
+public class SSLDebugLoggerUnitTest {
+    @Test
```

Comment:
Could we add a space

- Between the top of the class and the first line of code in the class (helps readability)
i.e.
public class SSLDebugLoggerUnitTest {

    @Test
...
}
- Between the test setup and assertions
i.e. 
```
...
 SSLDebugLogger.enableSSLDebugUsingSystemProperties();

 assertEquals("ssl:debug", System.getProperty("javax.net.debug"));
...
```

Reply:
Added a new line between class and test. for asserts statement I've grouped them into a logical way followed by a new line. Please review.